### PR TITLE
TCA-11 - TCA-11-create-a-readme-md

### PR DESCRIPTION
**Pull request by pscacchi**
The repository was created.
The README.m was created.
The .gitignore was created.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,152 +1,85 @@
...
+### Fork strategy: [Feature flow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
+ - Main branch: main
+ - Feature branch: feature/XXX-XXX-XXX-name-ticket
 
-
-* Roles:
-- * Podruct Owner
-- * Technical Lead
-- * Facilitator
-
```
Comment (sebastianMullerEndava) [inline README.md:13]: I think it is. not fork strategy is branch strategy xd

File: README.md [hunk 2]
```diff
@@ -1,152 +1,85 @@
...
-* Fork strategy
-- * [Feature flow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
-- * [Git flow](https://www.atlassian.com/es/git/tutorials/comparing-workflows/gitflow-workflow#:~:text=Gitflow%20es%20un%20modelo%20alternativo,vez%20y%20quien%20lo%20populariz%C3%B3)
-
```
Comment (sebastianMullerEndava) [inline README.md:17]: Maybe would be nice to put in the readme which one you will use.

Comment (pscacchi) [reply README.md:17]: ### Readme format

The format used to build this readme is  [GitHub Flavored Markdown Spec format](https://github.github.com/gfm/)

Comment (pscacchi) [reply README.md:17]: ### Readme format

The format used to build this readme is  [GitHub Flavored Markdown Spec format](https://github.github.com/gfm/)

File: README.md (hunk 1 above)
Comment (pscacchi) [reply README.md:13]: It was deleted.

File: README.md (hunk 2 above)
Comment (Diana-Endava) [reply README.md:17]: What @sebastianMullerEndava was referring is to only put in the readme the Git/Fork strategy that you are going to use for this project and not both as you did here

Comment (pscacchi) [reply README.md:17]: The following were added:
Fork strategy: [Feature flow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
Main branch: main
Feature branch: feature/XXX-XXX-XXX-name-ticket

Comment (sebastianMullerEndava) [reply README.md:17]: please change "fork strategy" to "branch strategy"

Comment (pscacchi) [reply README.md:17]: It's done

**Review by sebastianMullerEndava — APPROVED** [review]

**Review by Diana-Endava — APPROVED** [review]

**Pull request merged**